go-dlv.el

```diff
@@ -131,6 +132,19 @@ and source-file directory for your debugger."
   (setq paragraph-start comint-prompt-regexp)
   (run-hooks 'go-dlv-mode-hook))
 
+;;;###autoload
+(defun dlv-current-test ()
```

Comment:
For me using the function as it is implmented feels be more convenient:

- most of the time I use it to stop at the beginning of the test and then put an "interesting" breakpoint somewhere else in the code
- I don't need to position the cursor at the line where a valid breakpoint can be set

How about having two functions instead: dlv-func-at-point and dlv-at-point? The only difference between them would be a breakpoint location, current function (will work with tests too) or current line.